src/FlightDisplay/FlightDisplayView.qml

```diff
@@ -728,6 +729,133 @@ Item {
         }
     }
 
+    //-- Annunciator panel
+    Rectangle {
+        visible:                    _showAnnunciatorPanel
+        id:                         annunciatorPanel
+        anchors.horizontalCenter:   parent.horizontalCenter
+        anchors.topMargin:          _toolsMargin
+        anchors.top:                parent.top
+        z:                          _mapAndVideo.z + 4
+        color:                      qgcPal.globalTheme === QGCPalette.Light ? QGroundControl.corePlugin.options.toolbarBackgroundLight : QGroundControl.corePlugin.options.toolbarBackgroundDark
+        radius:                     ScreenTools.defaultFontPixelWidth / 2
+        width:                      annunciatorRow.width + _toolsMargin * 2
+        height:                     annunciatorRow.height + _toolsMargin * 2
+
+        Row {
+            id:     annunciatorRow
+            spacing:    ScreenTools.defaultFontPixelWidth
+            anchors.horizontalCenter: parent.horizontalCenter
+            anchors.verticalCenter: parent.verticalCenter
+
+            Repeater {
+                id: annunciatorRepeater
+                property var ekfLowThresh: 0.5
+                property var ekfHighThresh: 0.8
+                property var vibeLowThresh: 30
+                property var vibeHighThresh: 50
```

Comment:
Should these be facts vs hard coded?

Reply:
@ptran not sure how to "hard code" these.  I made them `readonly property double`s...is that what you mean?

Reply:
Oh, if you're asking why I didn't put them inline in the code: These might become parameters in the future that are modifiable by the user.  TBD.